db/seeds.rb

```diff
@@ -5,3 +5,64 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+users = User.create([ 
+  {username:"dj",
+   password_digest: "dj",
+   role: 1,
+   full_name: "Dj",
+   email: "djloveshisgf@gmail.com"},
+
+  {username: "fred",
+   password_digest: "blocknroll",
+   role: 1,
+   full_name: "Fred",
+   email: "fredrules@gmail.com"},
+
+  {username: "tino",
+   password_digest: "poop",
+   role: 1,
+   full_name: "Tino",
+   email: "fredrules@gmail.com"}])
+
+   breeds = Breed.create([
+     {name: "sabertooth",
+      retired: false,
+      description: "A tasty large fanged feline"},
+
+      {name: "prionailurus viverrinus",
+       retired: false,
+       description:"Lives: near marshes, mangroves, rivers and streams in India and Southeast Asia"},
```

Comment:
Line is too long. [101/80]<br>Space missing after colon.<br>Space inside } missing.